main.go

```diff
@@ -94,6 +95,8 @@ var (
 func init() {
 	var err error
 
+	version = "1.7.0"
```

Comment:
The value for this variable is filled during compilation time, why should we add a fixed version?

Reply:
:flayte_mode_on: 👀 

sucede que tengo la versión ahí, que la usa el `healthz` y también la tengo en el Makefile, entonces debo mantener la consistencia en ambos lugares. Se me ocurre tener la versión solo en en Makefile y pasar el valor por un flag hacia el programa.